eth/catalyst/api.go

```diff
@@ -183,6 +183,7 @@ func (api *ConsensusAPI) makeEnv(parent *types.Block, header *types.Header) (*bl
 }
 
 func (api *ConsensusAPI) GetPayloadV1(payloadID PayloadID) (*ExecutableDataV1, error) {
+	log.Trace("GetPayload", "payloadID", payloadID)
```

Comment:
Nitpick: I was logging `payloaid` as context tag elsewhere for consistency with previous logging. It would make sense to just update all logs to use `payloadID` really.

Reply:
I don't understand :see_no_evil: